src/arkit.cs

```diff
@@ -531,6 +534,7 @@ interface ARTrackable {
 	[iOS (11,0)]
 	[NoWatch, NoTV, NoMac]
 	[BaseType (typeof(ARConfiguration))]
+	[DisableDefaultCtor]
```

Comment:
double check headers/doc - I suspect it's available

Reply:
Yes https://developer.apple.com/documentation/arkit/arfacetrackingconfiguration/2928280-init?changes=latest_beta